meteor/server/api/playout.ts

```diff
@@ -1505,6 +1548,53 @@ export namespace ServerPlayoutAPI {
 
 		updateTimeline(runningOrder.studioInstallationId)
 	}
+	export const roToggleSegmentLineArgument = syncFunction(function roToggleSegmentLineArgument (roId: string, slId: string, property: string, value: string) {
```

Comment:
a toggle-function is not ideal, because that requires the caller to know something about the value of the property, before it's about to change it.
Could we remove the "toggle" functionality (and if you want to remove an runtime-argument, you pass the value null instead)?

Reply:
I guess, though this better maps to the action performed by the user - pretty much like roTake does various things depending on the holdState of the current segment line. The user just expects to have a single action that toggles a given argument. Is your argument that the current state check should be done against what is in the GUI, because that is the state that the user is actually toggling (as to what's server-side, which just might be something else)?